parsers/RU.py

```diff
@@ -115,13 +115,13 @@ def fetch_production_1st_synchronous_zone(
         raise NotImplementedError("This parser is not able to parse given zone")
 
     if target_datetime:
-        target_datetime_tz = arrow.get(target_datetime).to(tz)
+        target_datetime_tz = target_datetime.replace(tzinfo=TIMEZONE)
```

Comment:
The equivelant of arrows `.to()` is `.astimezone()`
```suggestion
        target_datetime_tz = target_datetime.astimezone(tz=TIMEZONE)
```